components/warn_metric_ack_modal/warn_metric_ack_modal.tsx

```diff
@@ -161,6 +162,28 @@ export default class WarnMetricAckModal extends React.PureComponent<Props, State
             />
         );
 
+        const learnMoreLink = 'https://mattermost.com/pl/default-admin-advisory';
+        const subText = (
+            <div
+                style={{display: 'flex', opacity: '0.56', flexWrap: 'wrap'}}
```

Comment:
would make more sense if this was just added to the class? or maybe a second class added to this with the above styles, something like `help__format-subtext`? 1/5 on it though doesnt really matter too much

Reply:
probably, though in practice we create these one-off css classes that are never re-used, so I chose to keep the overriding settings inline since they are not reused (similar to an anonymous class/inline const).